examples/data-sources/grafana_folders/data-source.tf

```diff
@@ -0,0 +1,16 @@
+resource "grafana_folder" "test_a" {
+  title = "test-folder-a"
+  uid   = "test-ds-folder-uid-a"
+}
+
+resource "grafana_folder" "test_b" {
+  title = "test-folder-b"
+  uid   = "test-ds-folder-uid-b"
+}
```

Comment:
Would it be preferable if the example used a single `grafana_folder.test` and a `for_each`, rather than declaring semi-redundant `grafana_folder` resources?

Reply:
All good like this